x-pack/plugins/apm/public/components/app/service_details/index.tsx

```diff
@@ -20,9 +20,7 @@ export function ServiceDetails({ match, tab }: Props) {
   return (
     <div>
       <ApmHeader>
-        <EuiTitle>
-          <h1>{serviceName}</h1>
-        </EuiTitle>
+        <ServiceIcons serviceName={serviceName} />
```

Comment:
Why is the `h1` subsumed inside of the `ServiceIcons` component? I assumed they would be separate things adjacent to one another.